client/boot/login.js

```diff
@@ -0,0 +1,28 @@
+/** @format */
+// Initialize polyfills before any dependencies are loaded
+import './polyfills';
+
+/**
+ * External dependencies
+ */
+import debugFactory from 'debug';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import loginModule from 'login';
+import createReduxStoreFromPersistedInitialState from 'state/initial-state';
+import * as controller from 'controller/index.web';
+import { setupContextMiddleware } from './common';
+
+const debug = debugFactory( 'calypso' );
+
+window.AppBoot = () => {
+	debug( 'boot login page' );
+	createReduxStoreFromPersistedInitialState( reduxStore => {
```

Comment:
we'll want a different version of this just for login with a different reducer tree and probably no persistence support